### DISCUSSION TITLE
CSV Import is not available on Live Demo (v10.0.x)

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#9445] [Q&A / Support] [answered]
Hello,
Not able to see CSV Import option in data object tree on Live Demo (v10.0.x).
Please guide if there are any settings that need to be done.

![image](https://user-images.githubusercontent.com/52573912/121994571-26e83100-cdc3-11eb-9d3f-5560497f32ce.png)

**Comment by weisswurstkanone** [comment, marked as answer]
CSV import has been removed in Pimcore 10.
See https://github.com/pimcore/pimcore/blob/f3f444fef81cd1d4cf4c4faae8e4efd869865a9b/doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md#L185

Use  https://github.com/pimcore/data-importer or https://github.com/w-vision/DataDefinitions instead.
